ci/dockerfiles/Dockerfile.ansible

```diff
@@ -0,0 +1,41 @@
+FROM osdk-builder as builder
```

Comment:
Nit: use `.Dockerfile` as a file extension, since its a file type. Also code editors pick up the file type that way.